data/kahet/kahet missions.txt

```diff
@@ -721,3 +721,1128 @@ event "pacified ka'sei"
+			`	As the crew moves beyond your line of sight and communications cease, your unease grows steadily. As you split your attention between the various screens in front of you while glancing out of the airlock, every minute seems to last an eternity.`
+			`	Scanning over the monitors for the umpteenth time, you see that heightened energy levels have been detected inside the unknown creature.`
+			choice
+				`	(Wait and see what happens.)`
+					goto waiting
+				`	(Shoot the Fetri'sei with my sidearm.)`
+			`	You take the weapon from the table and shoot the Fetri'sei just as the creature covers itself in rose-tinted sparks. According to Dusk's words, one shot should be enough to send the message through the impenetrable armor of the spacecraft; but nothing seems to happen after your gun's energy bolt disperses on the surface of the ship.`
+				goto fired
+			label waiting
+			`	A second later, the creature covers itself in a burst of rose-colored sparks: it's about to jump away! You take the weapon from the table and pull the trigger, releasing a bolt of energy that quickly disperses on the surface of the Fetri'sei. According to Dusk's words, it should be enough to send the message through the impenetrable armor of the spacecraft; but nothing seems to happen after the shot.`
+			label fired
+			`	Second after second passes relentlessly, with no reaction whatsoever from the Fetri'sei or its surroundings. As the unknown creature fades behind a violet cloud and jumps away, you can't help but wonder what to do.`
+			choice
+				`	(Shoot the Fetri'sei a second time.)`
+					goto shoot
+				`	(Keep waiting.)`
+					goto wait
+				`	(Jump on the Ka'sei to call the Remnant aboard.)`
+			`	There's no time to waste. Without hesitation, you push yourself off the border of the airlock and toward the Fetri'sei. After remaining airborne for a few instants, you land right next to the leftmost engine nacelle, bouncing off its skin. A timely activation of the thrusters on your wrists helps you move back to its surface, however, and by grabbing a long spike attached to its back, you can finally stabilize and move further ahead.`
+			`	White noise starts coming from your communicator; moving your head up, you finally see the Remnant leaping out of the hatch, carrying a dozen faintly glowing cylindrical tubes under their arms. They notice you right when communication returns to normal, and their message arrives loud and clear: "Jump!"`
+			`	You leave the handles and use your feet to give you a good push away from the spacecraft. At the last second, you move to the right to avoid ending up behind one of its thrusters right as they roar to life. The Fetri'sei has awakened, and for an instant you see all of its systems activate at once - the engines, the steering, even the Ravagers.`
```

Comment:
```suggestion
			`	You leave the handles and use your feet to give you a good push away from the spacecraft. At the last second, you move to the right to avoid ending up behind one of its thrusters just as they roar to life. The Fetri'sei has awakened, and for an instant you see all of its systems activate at once - the engines, the steering, even the Ravagers.`
```